components/TextFields/src/MDCTextInputControllerFullWidth.m

```diff
@@ -731,21 +731,21 @@ - (void)updateFontsForDynamicType {
  MAX(underlineLabelsOffset,MDCTextInputVerticalHalfPadding)           // Padding and/or labels
  */
 // clang-format on
-- (UIEdgeInsets)textContainerInset:(UIEdgeInsets)defaultInsets {
+- (UIEdgeInsets)textInsets:(UIEdgeInsets)defaultInsets {
   // NOTE: UITextFields have a centerY based layout. But you can change EITHER the height or the Y.
   // Not both. Don't know why. So, we have to leave the text rect as big as the bounds and move it
   // to a Y that works. In other words, no bottom inset will make a difference here for UITextFields
-  UIEdgeInsets textContainerInset = UIEdgeInsetsZero;
```

Comment:
In the future, this kind of search-and-replace is a nice candidate for a single, trivially-reviewed, PR.